### PR TITLE
modify image pullPolicy

### PR DESCRIPTION
**Pull request by zackzhangkai**
Signed-off-by: zackzhangkai <zhangkaiamm@gmail.com>

### CODE REVIEW

**Comment by zackzhangkai** [comment]
assign @pixiake

File: roles/ks-istio/templates/custom-values-jaeger.yaml.j2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 image:
   repository: {{ jaeger_operator_repo }}
   tag: {{ jaeger_operator_tag }}
-  pullPolicy: {{ ks_image_pull_policy }}
+  pullPolicy: IfNotPresent
```
Comment (LinuxSuRen) [inline roles/ks-istio/templates/custom-values-jaeger.yaml.j2:7]: Why do not use the variable `ks_image_pull_policy` anymore?

Comment (zackzhangkai) [reply roles/ks-istio/templates/custom-values-jaeger.yaml.j2:7]: Just ks component use the variable, such as ks-apiserver, etc.

Comment (LinuxSuRen) [reply roles/ks-istio/templates/custom-values-jaeger.yaml.j2:7]: Got you.

**Pull request merged**